tutorial/stage2/setup.py

```diff
@@ -1,31 +1,40 @@
 import sys
 from esky import bdist_esky
 from distutils.core import setup
-
-# for windows
+    
+# for freezing with esky
+# uncomment the block you want to run depending on your freezer
 # > python setup.py bdist_esky
-if sys.platform in ['win32','cygwin','win64']:
 
-    # Use bdist_esky instead of py2exe
+# Using py2exe
+#setup(
+#    name = "example-app",
+#    version = "0.2",
+#    scripts = ["example.py"],
+#    options = {"bdist_esky": {
+#              "freezer_module":"py2exe",
+#            }}
+#)
 
-    setup(
-        name = "example-app",
-        version = "0.2",
-        #  All executables are listed in the "scripts" argument
-        scripts = ["example.py"],
-        options = {"bdist_esky": {
-                  "freezer_module":"py2exe",
-                }}
-    )
+# Using py2app
+#setup(
+#    name = "example-app",
+#    version = "0.2",
+#    scripts = ["example.py"],
+#    options = {"bdist_esky": {
+#                "freezer_module":"py2app"
+#             }}
+#    )
```

Comment:
I wonder if we should break these out into separate subdirectories, one per freezer module?